packages/frontend/amp/components/elements/AudioAtom.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export const AudioAtom: React.FC<{
+    element: AudioAtomElement;
+}> = ({ element }) => {
+    return (
+        <amp-audio src={element.trackUrl} title={element.kicker}>
+            <div fallback="">
+                <p>Your browser doesn’t support HTML5 audio</p>
```

Comment:
We could possibly improve this?

    We're unable to serve this media because your browser does not support HTML 5 audio.

(or something like that)

Reply:
Updated. Thanks!